docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  wordpress:
+    image: wordpress:latest
+    container_name: wordpress
+    ports:
+      - "8080:80"
+    environment:
+      WORDPRESS_DB_HOST: db
+      WORDPRESS_DB_USER: wordpress
+      WORDPRESS_DB_PASSWORD: wordpress
```

Comment:
No Secret Info in repo! Document other handling possibilities and change this, please. Also, you should not use/rely on default or unsafe passwords like `password`, `<APP_NAME>`, or similar.